README.md

```diff
@@ -54,8 +54,8 @@ This should start the containers and allow you to setup the database.
 
 To initialize the database (create user, tables, etc.) run these commands:
 
-    $ docker-compose run web python manage.py init_db --skip-create
-    $ docker-compose run web python manage.py init_msb_db 
+    $ docker-compose run web python manage.py init_db 
+    $ docker-compose run web python manage.py init_msb_db --create-db
```

Comment:
I think we should work to close some of the open PRs are get armacolite's branch merged before we open the MSB can of worms. But yes, the current state of things is less than ideal.